### PR TITLE
Edit bash script to fit protocol change

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

File: src/System.CommandLine.Suggest/dotnet-suggest-shim.bash [hunk 1]
```diff
@@ -2,7 +2,8 @@
 _dotnet_bash_complete()
 {
     local fullpath=`type -p ${COMP_WORDS[0]}`
-    local completions=`dotnet-suggest get --executable "${fullpath}" --position ${COMP_POINT} -- ${COMP_LINE}`
+    local escaped_comp_line=$(echo "$COMP_LINE" | sed s/\"/'\\\"'/g)
```
Comment (wli3) [inline src/System.CommandLine.Suggest/dotnet-suggest-shim.bash]: Although escape in powershell doesn't seem to work

Comment (wli3) [reply src/System.CommandLine.Suggest/dotnet-suggest-shim.bash]: This line https://github.com/dotnet/command-line-api/blob/master/src/System.CommandLine.Suggest/dotnet-suggest-shim.ps1#L10

Comment (wli3) [reply src/System.CommandLine.Suggest/dotnet-suggest-shim.bash]: t-rex --file "c:/good"

with log
```
dotnet-suggest received: |get|-e|C:\Users\wul\.dotnet\tools\t-rex.exe|--position|22|--|t-rex --file "c:/good"
```

**Comment by jonsequitur** [comment]
@dotnet-bot test this please

**Comment by wli3** [comment]
oops i directly checked in the remove extra line in yaml. i will watch out next time

File: src/System.CommandLine.Suggest/dotnet-suggest-shim.bash (hunk 1 above)
Comment (jonsequitur) [reply src/System.CommandLine.Suggest/dotnet-suggest-shim.bash]: Not sure I understand the problem here but it sounds like it's something to open as a separate issue.
 
Is this PR ready to merge?

**Comment by wli3** [comment]
this is good to go

**Pull request merged**